pkg/deploy/deploy.go

```diff
@@ -59,12 +61,19 @@ type DeployConfigsOptions struct {
 }
 
 var (
-	lock sync.Mutex
+	lock                         sync.Mutex
+	concurrentDeploymentsLimiter *rest.ConcurrentRequestLimiter
 
 	skipError = errors.New("skip error")
 )
 
 func Deploy(ctx context.Context, projects []project.Project, environmentClients dynatrace.EnvironmentClients, opts DeployConfigsOptions) error {
+	maxConcurrentDeployments := environment.GetEnvValueIntLog(environment.ConcurrentDeploymentsEnvKey)
```

Comment:
Should we move this to a `func init` block? It is a global thing, and I don't think the initialization here is the best.

I'd love to have it passed down in the functions to have it per function call, but that would mean more changes :/

Reply:
I did try it in an init block, but then the logs were first, which I don't like. (I would like the logs, just to make it easy for us to identify if it is being used / causing problems)

We could pass it around via context, what do you think? Then it wouldnt be global, so could be initialized here in `Deploy(...)`